src/main/java/org/apache/sysml/api/mlcontext/ScriptExecutor.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.sysml.api.DMLScript;
 import org.apache.sysml.api.ScriptExecutorUtils;
+import org.apache.sysml.api.DMLScript.DMLOptions;
```

Comment:
I would say that DMLOptions is so important to SystemML that it should probably be moved to its own class.